app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -332,7 +334,10 @@ public void onClick(DialogInterface dialog, int which) {
                 // Place a call
                 EditText contact = (EditText) ((AlertDialog) dialog).findViewById(R.id.contact);
                 twiMLParams.put("to", contact.getText().toString());
-                activeCall = Voice.call(VoiceActivity.this, accessToken, twiMLParams, callListener);
+                ConnectOptions connectOptions = new ConnectOptions.Builder(accessToken)
+                        .params(twiMLParams)
```

Comment:
Let's rename `twiMLParams` to `params`. I think this may clarify the distinction between what is passed the TwiML application vs what is used in the twiml helper libraries themselves.

Reply:
ok